release-1.8/release_notes_draft.md

```diff
@@ -165,68 +165,43 @@ across the system. Here's the release [scalability validation report].
 
-* The following deprecated flags have been removed from `kube-controller-manager`:
+* The following deprecated flags were removed from `kube-controller-manager`:
 
   * `replication-controller-lookup-cache-size`
   * `replicaset-lookup-cache-size`
   * `daemonset-lookup-cache-size`
 
-  Do not use deprecated flags.
-
-* StatefulSet: The deprecated `pod.alpha.kubernetes.io/initialized` annotation for
-  interrupting StatefulSet Pod management is now ignored. If the annotation is set
-  to `true` or left blank, you won't see any change in behavior. If it's set to
-  `false`, previously dormant StatefulSets might become active after upgrading.
-
-* CronJobs has been promoted to `v1beta1` which is now turned on by default.
-  Although version `v2alpah1` is still available, it is deprecated.  Migrate to
-  `batch/v1beta1.CronJobs`.  Additionally, upgrading cluster in high-availability
-  configuration might return errors. The new controllers rely on the latest
-  version of the resources.  If the expected version is not found during rolling
-  upgrade, the system throws resource not found errors.
-* The `batch/v2alpha1.ScheduledJobs` has been removed.  Migrate to `batch/v1beta.CronJobs`
-  to continue managing time based jobs.
-
-* The APIs `rbac/v1alpha1`, `settings/v1alpha1`, and `scheduling/v1alpha1` are
-  disabled by default.
-
-* The `system:node` role is no longer automatically granted to the `system:nodes`
-  group in new clusters. It is recommended that nodes be authorized using the
-  `Node` authorization mode instead. Installations that wish to continue giving
-  all members of the `system:nodes` group the `system:node` role (which grants
-  broad read access, including all secrets and configmaps) must create an
-  installation-specific `ClusterRoleBinding`. ([#49638](https://github.com/kubernetes/kubernetes/pull/49638))
+  Don't use these flags. Using deprecated flags causes the server to print a warning. Using a removed flag causes the server to abort the startup.
+
+* StatefulSet: The deprecated `pod.alpha.kubernetes.io/initialized` annotation for interrupting the StatefulSet Pod management is now ignored. If the annotation is set to `true` or left blank, you will not see any change in behavior. If the annotation is set to `false`, previously dormant StatefulSets might become active after upgrading.
+
+* CronJobs was moved to `v1beta1` and is turned on by default. The deprecated `v2alpah1` CronJobs is still available, but it must be explicitly turned on. It's recommended that you migrate the current CronJobs to `batch/v1beta1.CronJobs`. The version changes might introduce errors during cluster upgrades in high-availability configuration. The new controllers rely on `v1beta1` of the resources during rolling upgrade. If the expected version is not found, the system throws resource not found errors.
+
```

Comment:
This item is hard to edit, I know, but I might reword the first part of the para to something like this:

The CronJob object is now enabled by default at `v1beta1`. CronJob `v2alpha1` is still available, but it must be explicitly enabled. We recommend that you move any current CronJob objects to `batch/v1beta1.CronJob`.

And then the rest of the para seems like it should be separated out, possibly into a warning sub-para in the listitem. It's also still a bit confusing even as re-written. ("Might introduce errors" is too vague. Also does the sentence about new controllers refer to all resources? Or just to the CronJob objects?) Assuming the latter, how about something like:

Be aware that if you specify the deprecated version, you may encounter Resource Not Found errors. These errors occur because the new controllers look for the new version during a rolling update.

Reply:
I checked with @soltysh this morning, he said both are closely related, so I rephrased that way.